assets/src/js/admin/onboarding-wizard/app/steps/your-cause/index.js

```diff
@@ -19,17 +22,23 @@ const YourCause = () => {
 		<div className="give-obw-your-cause">
 			<h2>{ __( 'What does fundraising look for you?', 'give' ) }</h2>
 			<CardInput values={ userType } onChange={ ( values ) => setUserType( values ) } checkMultiple={ false } >
-				<Card value="testing">
-					<h1>Individual</h1>
+				<Card value="individual" padding="60px 32px">
```

Comment:
If the padding is always the same, then it should be extracted to the `<Card />` component definition and applied with a stylesheet.

Reply:
Per my previously reply, I'm using a `padding` prop to pass padding rules to Cards on a card-by-card basis because the Figma designs are inconsistent with regards to card padding from step to step. But with your suggestion in mind, because each card in this step does use the same padding, I opted to use a cardPadding const for the step, to avoid having loose strings. See ref: https://github.com/impress-org/givewp/commit/2d55320da4a5f2a3cc7fb3af2def2842fd958701